### PR TITLE
simple grid overlay added

### PR DESCRIPTION
**Pull request by derFogel**
I added an qml only grid overlay. It is configured with a ComboBox in the general settings view.
I also added added qsTr to "switch zoom control".

### CODE REVIEW

File: qml/pages/GridOverlay.qml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+
+Item {
+    visible: settings.global.gridMode != "none"
+
+    function horizontalLines(id) {
+        if (id === "thirds") return [0.33, 0.66];
+        if (id === "ambiente") return [0.21, 0.79];
```
Comment (piggz) [inline qml/pages/GridOverlay.qml]: this doesnt work ... i think because further down its called ambience

**Comment by piggz** [comment]
When you fix the issue, please git commit --amend and force push to keep the history clean

**Comment by derFogel** [comment]
done, I also fixed a mistake in the German translation.

PS: I think you can close issue number 22, 24 and 31 (after merge)

**Comment by piggz** [comment]
One thing i was wondering, should the lines be placed relative to the whole view, or to the resolution, or viewfinder size?

**Comment by piggz** [comment]
@derfogel, discussed with mal, and we think that jolla camera does take the aspect ratio into account when drawing the lines.  do you think you can add that?

**Comment by derFogel** [comment]
Done.

**Comment by derFogel** [comment]
Is there a reason why this is not merged? I added the resolution feature about a week ago.

**Comment by piggz** [comment]
Yes, i forgot to test!

File: qml/pages/SettingsOverlay.qml [hunk 2]
```diff
@@ -233,13 +233,45 @@ Item {
...
                         checked: settings.global.swapZoomControl
                         onCheckedChanged: {
                             settings.global.swapZoomControl = checked;
                         }
                     }
+
+                    ComboBox {
+                        id: gridSwitch
+
+                        property var grids: [{"id": "none", "name": qsTr("none")}, {"id": "thirds", "name": qsTr("thirds")}, {"id": "ambience", "name": qsTr("ambience")}]
```
Comment (piggz) [inline qml/pages/SettingsOverlay.qml]: Can you capitalise the visible strings

File: qml/pages/GridOverlay.qml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    function verticalLines(id) {
+        if (id === "thirds") return [0.33, 0.66];
+        if (id === "ambience") return [0.2333, 0.7666];
+        return [];
+    }
+
+    Repeater {
+        model: horizontalLines(settings.global.gridMode)
+
+        delegate: Rectangle {
```
Comment (piggz) [inline qml/pages/GridOverlay.qml]: Would it be possible to add a gradient fade like the stock camera?

**Comment by derFogel** [comment]
I added the opacity effect. It wasn't quite easy because a horizontal gradient is that easy in Qt 5.6. I used a OpacityRampEffect to archive this.

**Pull request merged**